src/components/BackToTop.js

```diff
@@ -12,7 +12,7 @@ export const BackToTop = ({ onPress }) => {
     <TouchableOpacity onPress={onPress}>
       <Wrapper>
         <Icon.ArrowUp style={styles.icon} />
-        <BoldText link style={styles.backToTop}>
+        <BoldText center primary style={styles.backToTop}>
```

Comment:
i have adjusted this accordingly to the new placeholder component and set the text for the placeholder "button" to uppercase as it is here.